detail/provisioning.cattle.io.cluster.vue

```diff
@@ -73,12 +73,15 @@ export default {
       hash.normanNodePools = this.$store.dispatch('rancher/findAll', { type: NORMAN.NODE_POOL });
     }
 
+    hash.kubeNodes = this.$store.dispatch('cluster/findAll', { type: NODE });
```

Comment:
As per PR comment, this can be called only for RKE2 clusters

Reply:
Fixed by only fetching `kubeNodes` when we fetch the collection of machines